datalad/interface/base.py

```diff
@@ -110,6 +110,11 @@ def alter_interface_docs_for_cmdline(docs):
         '',
         docs,
         flags=re.MULTILINE | re.DOTALL)
+    # remove :term:`...` RST markup for cmdline docs
+    docs = re.sub(
+        r':term:`.*`[\\]*',
```

Comment:
`*` is greedy so would cause incorrect matching with two `:term:` are found in the same line.  I think it is worth replacing with `r':term:`[^`]*`[\\]*'`